parser_test.go

```diff
@@ -1196,6 +1197,8 @@ func TestParserParse(t *testing.T) {
 
 		// Make sure the json remains valid after visiting all the items.
 		ss := v.String()
+		s = strings.ReplaceAll(s, "&", "\\u0026") // there is a '&' in the json, which should be escaped.
```

Comment:
HTML escaping by default is consistent with both `encoding/json` _and_ jsoniter. However, `fastjson` evidently hasn't been doing that up until now (right?). Should we take the time to look into the codebase and check whether this could cause any issue?

Reply:
> However, fastjson evidently hasn't been doing that up until now (right?)

Right. As far as I can tell. 

> Should we take the time to look into the codebase and check whether this could cause any issue?

We will, in the internal PRs, but If something, it should _solve_ issues. 😄